extensions/data-transfer/portability-data-transfer-microsoft/src/main/java/org/datatransferproject/transfer/microsoft/photos/MicrosoftPhotosImporter.java

```diff
@@ -140,7 +140,9 @@ public ImportResult importItem(
   private String createOneDriveFolder(PhotoAlbum album) throws IOException {
 
     Map<String, Object> rawFolder = new LinkedHashMap<>();
-    rawFolder.put("name", album.getName());
+    // clean up album name for microsoft specifically
+    String albumName = Strings.isNullOrEmpty(album.getName()) ? "Untitled" : album.getName();
```

Comment:
I think you need to trim it first to get rid of the whitespace. this would return false for a string with a single space.

Reply:
no, it gets trimmed in PhotoAlbum to remove trailing and leading spaces.

Reply:
Ah forgot about that, lets add it in the above comment.